src/main/kotlin/rx/lang/kotlin/subjects.kt

```diff
@@ -1,13 +0,0 @@
-package rx.lang.kotlin
```

Comment:
Because I think these functions masquerading as constructors are misleading. Maybe the serialized method makes sense though.